### ISSUE TITLE
Error in flow >= 0.34.0

### ISSUE THREAD

**Issue by tkuminecz** [andreypopp/validated#8]
Hi,

Using Flow v0.34.0 or v0.35.0 I get the following error:

```
node_modules/validated/lib/schema.js.flow:228
228:   extends Node<$ObjMap<S, <V>(v: Node<V>) => V>> { // eslint-disable-line no-undef
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ object type. This type is incompatible with
 28:   context: Context;
                ^^^^^^^ Context
```

However, the error is not present If I revert back to v0.33.0.

**Comment by andreypopp** [comment]
Sorry it took too long. Fixed it as of 0.8.4.

**Comment by tkuminecz** [comment]
No problem, and thanks for the great work!
